web/react/stores/preference_store.jsx

```diff
@@ -122,5 +122,6 @@ class PreferenceStoreClass extends EventEmitter {
 }
 
 const PreferenceStore = new PreferenceStoreClass();
+PreferenceStore.setMaxListeners(0);
```

Comment:
Preferably have the PostsView doing the event listening (and only when active) and then just pass what you need in the posts down through props. The idea is (which haven't implemented very well yet) is to have view controllers which are listening to the stores. These controllers would then pass the updated store data downwards through props to any children that need it.